Gopkg.toml

```diff
@@ -29,7 +29,7 @@ required = ["github.com/3rf/mongo-lint"]
 
 [[constraint]]
   name = "github.com/mongodb/mongo-tools-common"
-  version = "v3.0.8"
+  revision = "c247ad5cc1fb30d566fca9f1f1a72440553845dd"
```

Comment:
I'll create a new mtc tag after merging that [PR](https://github.com/mongodb/mongo-tools-common/pull/39)!